sdk/core/Azure.Core.Experimental/src/Primitives/README.md

```diff
@@ -0,0 +1,55 @@
+# Binary Data
```

Comment:
This is a strange file location.

Reply:
I think we can move it into the Core readme eventually.

Reply:
Why not put it into Azure.Core.Experimental readme and get it close to something that we would have in Azure.Core readme.